### ISSUE TITLE
Specify regions/domains to use for superpositions

### ISSUE THREAD

**Issue by sillitoe** [UCLOrengoGroup/cath-tools#1]
It would be really useful to be able to superpose large, multi-domain structures based on just a single, shared domain. Equally, it might be useful to superpose structures on the residues around a specific set of important residues (e.g. active site, surface patch).

**Comment by tonyelewis** [comment]
Note to self: remember to sort out colourings so they do something sensible when using `superposition_content_spec`.

**Comment by tonyelewis** [comment]
Note to self: these are useful PyMOL commands when looking at testcase [1bdh](http://www.ebi.ac.uk/pdbe/entry/pdb/1BDH):

~~~~~
color black
show_as cartoon
show_as dots, hetatm
show_as sticks, organic
select 1bdhA01, ////3-59/
select 1bdhA02, ////60-161+292-323/
select 1bdhA03, ////162-291+324-340/
color blue,  1bdhA01
color red,   1bdhA02
color green, 1bdhA03
deselect
~~~~~

**Comment by tonyelewis** [comment]
This was implemented in 9fb416419dadeb91a7de0776207100e6a79c2ee9, which adds the following options block to `cath-superpose`:

~~~~~
Segment overlap/removal:
  --align-regions <regions>                Handle region(s) <regions> as the alignment part of the structure.
                                           May be specified multiple times, in correspondence with the structures.
                                           Format is: D[5inwB02]251-348:B,408-416A:B
~~~~~

It'd be useful if `cath-ssap` also allowed users to specify alignment regions. I'll open a new issue for that.
